launch/simulator/simulator.launch

```diff
@@ -8,7 +8,7 @@
     <arg if="$(arg gdb)" name="launch-prefix" value="gdb -ex run --args"/>
     <arg unless="$(arg gdb)" name="launch-prefix" value=""/>
 
-    <arg name="env" default="$(optenv ROBOT_ENV robotics_testlab_A)" />
```

Comment:
Why? It is a double PR. I know it should not be the case, but it is clear in the title.